### ISSUE TITLE
Support gohtml

### ISSUE THREAD

**Issue by sybrenstuvel** [Glavin001/atom-beautify#1190]
# Description

atom-beautify currently (I tried version doesn't support files with grammar "HTML (Go)" and extension "gohtml". It would be great if it did!
# Expected Results

Here is an example of such a file:

``` gohtml
{{define "footer"}}
  <footer>
    <p>
      Copyright 2016 je moeder
    </p>
    <p>
      Contact information: <a href="mailto:je@moeder">je@moeder</a>.
    </p>
  </footer>
{{end -}}

{{define "widget"}}
<div class="widget">
    {{template "widget-header" .Name}}
    <span class="price">€{{.Price}}</span>
    <pre>{{.}}</pre>
</div>
{{- end -}}

{{define "widget-header" -}}
    <h3 class="name">{{.}}</h3>
{{- end -}}

<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title>Go tutorials</title>
    <style>
        html {
            font-family: sans-serif;
            color: #ddd;
            background-color: #333;
        }
        body {
            margin-left: auto;
            margin-right: auto;
            width: 70ex;
            max-width: 100%;
        }
    </style>
</head>
<body>
<h1>Hello, {{if .Name -}} {{.Name}} {{- else -}} there {{- end}}!</h1>

<p>ViewData {{.}}</p>
<p id='viewdata'></p>

<script>
    var elt = document.getElementById('viewdata');
    elt.textContent = JSON.stringify({{.}});
</script>

{{range .Widgets}}
    {{- template "widget" . -}}
{{end}}

{{template "footer"}}
</body>
</html>
```
# Steps to Reproduce
1. Possibly use atom-beautify 0.29.11 on Atom 1.10.2, running on Kubuntu Linux 16.04. That's what I did, anyway.
2. Add code to Atom editor
3. Run command `Atom Beautify: Beautify Editor`
4. Get a message "Atom Beautify could not find a supported beautifier for this file"
# Checklist
- [x] I have tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] I have reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched for existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
  so I know this is not a duplicate issue
- [ ] Not done: Generated debugging information and added link for `debug.md` Gist to this issue, as this isn't a bug report but a feature request. And also because it couldn't be generated (see https://github.com/Glavin001/atom-beautify/issues/482#issuecomment-246300438)

**Comment by prettydiff** [comment]
Where can I find documentation for this?  I need to know the difference between these:

    {{- end -}}
    {{- end}}
    {{end -}}
    {{end}}

**Comment by hocine** [comment]
thanks, @prettydiff !

{{- end }} trims whitespace to the left of the {{ end }}

the docs are here [0] [1]

[0] https://golang.org/pkg/text/template 
[1] https://golang.org/pkg/html/template

**Comment by s4l1h** [comment]
What is status?

**Comment by prettydiff** [comment]
@s4l1h I need to get the latest version of Pretty Diff into Atom Beautify.  I am working on a more stable alternative to NPM, but have hit a snag.  I could really use your help: https://github.com/prettydiff/biddle/issues/25#issuecomment-270567516

**Comment by s4l1h** [comment]
@prettydiff added.But very old package `"prettydiff": "^1.16.27",`

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1466 has been merged.

**Comment by Glavin001** [comment]
Published to v0.29.19
